README.md

```
@@ -1,14 +1,9 @@
-# 🌐 E-Commerce Platform by Binary Brains
+# 🛒 AmarTech - An Ecommerce Platform
 
-<p align="center">
-  <a href="#">
-    <img src="https://img.shields.io/badge/Live_Site-Visit_Now-brightgreen?style=for-the-badge&logo=internet-explorer" alt="Live Site" />
-  </a>
-</p>
+AmarTech is a robust E-Commerce Web Application built with ASP.NET Core MVC, showcasing key features of a modern online store. Created for educational purposes, it adheres to clean architecture principles, demonstrating how to develop a scalable and maintainable system from the ground up.
 
-👉 **[Click here to visit the live site]()**
+## 📚 Explore Project Resources, management
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: img

(MD033, no-inline-html)